dev/doc/INSTALL.make.md

```diff
@@ -187,58 +169,28 @@ and coqc selects a particular binary.
   directory, or read online on http://coq.inria.fr/doc/)
   and in the corresponding manual pages.
 
-Compiling For Different Architectures.
---------------------------------------
-
-This section explains how to compile Coq for several architecture, sharing
-the same sources. The important fact is that some files are architecture
-dependent (`.cmx`, `.o` and executable files for instance) but others are not
-(`.cmo` and `.vo`). Consequently, you can :
-
--  save some time during compilation by not cleaning the architecture
-   independent files;
-
--  save some space during installation by sharing the Coq standard
-   library (which is fully architecture independent).
-
-So, in order to compile Coq for a new architecture, proceed as follows:
-
-* Omit step 7 above and clean only the architecture dependent files:
-  it is done automatically with the command
-
-        make archclean
-
-* Configure the system for the new architecture:
-
-        ./configure <options>
-
-  You can specify the same directory for the standard library but you
-  MUST specify a different directory for the binaries (of course).
-
-* Compile and install the system as described in steps 5 and 6 above.
-
 Moving Binaries Or Library.
 ---------------------------
 
-If you move both the binaries and the library in a consistent way,
-Coq should be able to still run. Otherwise, Coq may be "lost",
-running "coqtop" would then return an error message of the kind:
+If you move both the binaries and the library in a consistent way, Coq
+should be able to still run. Otherwise, Coq may not be able to find
+the required prelude files and thenreturn an error message of the
+form:
```

Comment:
```suggestion
If you move both the binaries and the library in a consistent way, Coq
should still be able to run. Otherwise, Coq may not be able to find
the required prelude files and will give this error message:
```